src/test/java/com/yahoo/bullet/dsl/converter/AvroBulletRecordConverterTest.java

```diff
@@ -37,26 +37,50 @@
                     .fields()
                         .name("string").type().optional().stringType()
                         .name("int").type().optional().intType()
                         .name("mapOfString").type().optional().map().values().stringType()
+                        .name("recordOfString").type().optional().record("myRecord").fields()
+                            .name("firstField").type().nullable().stringType().noDefault()
+                            .name("secondField").type().nullable().stringType().noDefault()
+                        .endRecord()
                         .name("mapOfDouble").type().optional().map().values().doubleType()
                         .name("mapOfMapOfString").type().optional().map().values().map().values().stringType()
+                        .name("mapOfRecordOfString").type().optional().map().values()
+                            .record("innerRecordInMap").fields()
+                                .name("firstField").type().nullable().stringType().noDefault()
+                                .name("secondField").type().nullable().stringType().noDefault()
+                            .endRecord()
+                        .name("recordOfRecordOfString").type().optional().record("myRecordOfRecord").fields()
+                            .name("firstField").type().optional().record("innerRecordInRecord").fields()
+                                .name("firstField").type().nullable().stringType().noDefault()
+                                .name("secondField").type().nullable().stringType().noDefault()
+                            .endRecord()
+                        .endRecord()
                         .name("listOfString").type().optional().array().items().stringType()
                         .name("listOfBoolean").type().optional().array().items().booleanType()
                         .name("listOfMapOfString").type().optional().array().items().map().values().stringType()
+                        .name("listOfRecordOfString").type().optional().array().items().record("stringRecord").fields()
+                            .name("firstField").type().nullable().stringType().noDefault()
+                            .name("secondField").type().nullable().stringType().noDefault()
+                        .endRecord()
                         .name("unsupportedUnion").type().unionOf()
                             .nullType().and()
-                            .record("record")
-                            .fields()
-                                .name("stringField").type().optional().stringType()
-                                .endRecord()
+                            .enumeration("enumeration")
+                                .symbols("one", "two", "three")
                             .endUnion()
                             .nullDefault()
                         .endRecord();
+
+        RECORD_SCHEMA = SchemaBuilder.record("recordOfString")
+                    .fields()
```

Comment:
Indentation like SCHEMA above

Reply:
This is indented like SCHEMA above.

Reply:
Not the indentation for fields and then subsequent ones